src/ext/respToRedisLoader.c

```diff
@@ -0,0 +1,243 @@
+#include "common.h"
+#include <string.h>
+#include <unistd.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <errno.h>
+#include "readerResp.h"
```

Comment:
since this PR contains a file rename commit, and a refactor rename of its contents, it may be a good idea not to squash-merge it. (it looks like git isn't able to show it as a rename when looking at the whole diff)
in which case maybe you'll want to rearrange the commits to be cleaner before merging

Reply:
Sure. Thanks.